lib/utils/OpenFoodAPIConfiguration.dart

```diff
@@ -13,6 +14,9 @@ class OpenFoodAPIConfiguration {
   ///Defines a global userAgent to tell the backend the source of the request.
   static UserAgent? userAgent;
 
+  ///Defines a global user to avoid adding it to every request
+  static User? globalUser;
```

Comment:
what is this initialized with ?

Reply:
`Null`, we don't want to pass a user if there is non provided

Reply:
but I don't see `globalUser` getting any value, will it always be `Null`? Sorry if I'm missing something.

Reply:
The OpenFoodAPIConfiguration class is a kind of settings, that means the individual values can be set once at the start of the app and they are then automatically added to each requests. This includes things like the UserAgent or the individual url parts so that they don't have to be added to each request.

Reply:
Ah I see. In that case, I would advice making these private members and setting them via a constructor since you have methods for accessing them anyways.

The issue that can happen in current code is someone accesses `globalUser` directly instead of calling `getUser()` method

Reply:
Thats actually not needed here, they are allowed to be directly accessed for setting them and (maybe if someone wants to get them). We also allow to pass some of these configuration options via the method itself and we needed some kind of hirach. 

Most of my requests should go under the main account
             - But this special account should be send by a different user Account

And the method is only to simplify this process

Reply:
ok sounds good.